### ISSUE TITLE
A workaround to install vedo in Ubuntu 20.04

### ISSUE THREAD

**Issue by s1291** [marcomusy/vedo#152]
Ubuntu 20.04 is shipped with Python 3.8.2 but the` vtk` library is not yet available in PyPI for python 3.8 hence it not possible yet to install it using pip. Since` vtk` is the main dependency of `vtkplotter` it will not be installed.

The workaround, I found is to install `vtk7` using `apt` package manager:
```
sudo apt install python3-vtk7
```
clone the `vtkplotter` library:

```
git clone https://github.com/marcomusy/vtkplotter.git
```
in the file `setup.py` remove` vtk `from the dependencies. Change the line 15 to the following:
```
install_requires=[],
```
Now you can install vtkplotter as follows:
```
python setup.py install
```

I hope this is helpful.

**Comment by marcomusy** [comment]
Actually `vtkplotter` is in the debian repository!
So if you do 
`sudo apt install python3-vtkplotter`
you should get version 2020.2.0 along with vtk7

(i never tested it)

**Comment by RubendeBruin** [comment]
There is a wheels package available ( vtk-9.0.0-cp38-cp38-linux_x86_64.whl ) on the vtk download site. https://vtk.org/download/  Doesn't that work? (no idea why it is not on pypi yet)
